setup.py

```diff
@@ -20,6 +20,7 @@
     "coveralls",
     "mock",
     "pytz",
+    "psycopg2-binary",
```

Comment:
Why this is a graphene dependency?

Reply:
It's under "test_requires" but I am shy about adding this as well.

Reply:
I've just been following the CONTRIBUTING guide starting from a fresh install, within a virtual environment:

```bash
git clone git@github.com:graphql-python/graphene-django.git
cd graphene-django
virtualenv venv
source venv/bin/activate
make dev-setup
# Fix `DJANGO_SETTINGS_MODULE` in setup.cfg
make tests
```

yields:

```txt
FAILED graphene_django/tests/test_query.py::test_should_query_postgres_fields - ModuleNotFoundError: No module named 'psycopg2'
== 1 failed, 308 passed, 5 skipped, 15 warnings in 5.27s ==
```

I guess either `psycopg2` is really required, or this test should be skipped by default?

Reply:
Interesting, looks like a regression with Django 3.1 as it provides JSONField but not RangeField, so it no longer skips the test if postgres isn't installed. The test should check if RangeField is missing instead. You should definately drop the dependency. If you fix the test decorator that would be a bonus, or we can take care of it in a separate PR if you wish. Thank you for trying the guide and giving feedback.

Reply:
Alright! Please check :)